src/test/java/TestAPI.java

```diff
@@ -0,0 +1,154 @@
+import org.json.simple.JSONObject;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.TestInstance;
+
+import java.io.IOException;
+
+import static io.restassured.RestAssured.*;
+import static org.hamcrest.Matchers.equalTo;
+
+@TestInstance(TestInstance.Lifecycle.PER_CLASS)
+public class TestAPI {
+    private String baseURI;
+    private String ADDRESS_ENDPOINT;
+    private String USER_ENDPOINT;
+    private String PAYMENT_CARD_ENDPOINT;
+    private String userTable;
+    private String addressTable;
+    private String paymentCardTable;
+
+    private DatabaseController db;
+
+    @BeforeAll
+    public void initialize() throws IOException {
+        ConfigLoader properties = new ConfigLoader();
+        baseURI = properties.getPropertyValue("baseURI");
+        ADDRESS_ENDPOINT = properties.getPropertyValue("ADDRESS_ENDPOINT");
+        USER_ENDPOINT = properties.getPropertyValue("USER_ENDPOINT");
+        PAYMENT_CARD_ENDPOINT = properties.getPropertyValue("PAYMENT_CARD_ENDPOINT");
+        userTable = properties.getPropertyValue("userTable");
+        addressTable = properties.getPropertyValue("addressTable");
+        paymentCardTable = properties.getPropertyValue("paymentCardTable");
+        String dbURL = properties.getPropertyValue("dbURL");
+        String dbUsername = properties.getPropertyValue("dbUsername");
+        String dbPassword = properties.getPropertyValue("dbPassword");
```

Comment:
you don't really need to initialize this in @BeforeAll, simply do it in variable declaration